### ISSUE TITLE
Make compatible with composer v1

### ISSUE THREAD

**Issue by chillu** [silverstripe/silverstripe-graphql-composer-plugin#1]
If we don't do this, it effectively blocks anyone from using this plugin (and `silverstripe/graphql:4`) on Silverstripe's commercial platforms (CWP and Silverstripe Cloud). The upgrade to composer v2 for those platforms is planned, but ideally we don't create a two-way dependency there.

**Comment by maxime-rainville** [comment]
From memory, SCP/CWP block usage of plugins altogether. So making it composer-v1-compatible might not be enough.

**Comment by chillu** [comment]
No, they run with `--no-script`, but without `--no-plugins` - I've confirmed this with TSP.
